image/image.go

```diff
@@ -772,16 +772,19 @@ func setBootvars(downloadedSnapsInfoForBootConfig map[string]*snap.Info, model *
 	return nil
 }
 
-func extractKernelAssets(snapPath string, info *snap.Info) error {
+func extractKernelAssets(snapPath string, info *snap.Info, model *asserts.Model) error {
```

Comment:
Does this need to take the new model context thing or is a plain model sufficient? What if this code gets executed during remodelling?